README.md

```diff
@@ -72,7 +72,7 @@ To run Zilliqa, we recommend the following minimum system requirements:
     sudo apt-get install git libboost-system-dev libboost-filesystem-dev libboost-test-dev \
         libssl-dev libleveldb-dev libjsoncpp-dev libsnappy-dev cmake libmicrohttpd-dev \
         libjsonrpccpp-dev build-essential pkg-config libevent-dev libminiupnpc-dev \
-        libprotobuf-dev protobuf-compiler libcurl4-openssl-dev
+        libprotobuf-dev protobuf-compiler libcurl4-openssl-dev xml-twig-tools
```

Comment:
Can we be conservative when adding new dependencies? This is actually not a code build dependency and could be avoided by using some workaround. Also, you have to check whether `xml_grep` is available on Mac or not

Reply:
OK, I'll try a workaround without using this external tool